verify.bash

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+travis_tag=$1
+
+release_version_regex="^[0-9]+\\.[0-9]+\\.[0-9]+$"
+
+# Test: on release, we don't display the github url
+if [[ ${travis_tag} =~ ${release_version_regex} ]]
+then
+  if grep -q '^repo_url:' mkdocs.yml
```

Comment:
putting 'repo_url' in the config makes the github link appear in the upper right corner of the page.
I want that pre-release, not in the release.
I do think we should automate that: that release becomes an executor that removes that line; tags; puts it back.